model/src/main/scala/explore/model/ExploreSiderealTarget.scala

```diff
@@ -21,6 +26,22 @@ final case class SiderealTarget(id: SiderealTarget.Id, name: String, track: Prop
 object SiderealTarget {
   type Id = UUID
 
+  def New[F[_]: Sync]: F[SiderealTarget] =
```

Comment:
Seems odd to see a capitalized function name

Reply:
Yes, I will change this. This started life as a `val` constant, but it then evolved and I should have renamed it.